retrieval_service/datastore/providers/neo4j_graph_test.py

```diff
@@ -174,3 +175,60 @@ async def test_total_similar_to_relationships_count(ds: neo4j_graph.Client):
     assert (
         count == expected_count
     ), f"Expected {expected_count} BELONGS_TO relationships, but found {count}"
+
+
+amenities_search_test_data = [
+    pytest.param(
+        # "Where can I look for luxury goods?"
+        amenities_query_embedding2,
+        None,  # similarity threshold value
+        2,  # top_k value
+        [
+            {
+                "source_name": "Gucci Duty Free",
```

Comment:
Does this test ensure that the vector search is working ie would this test work just using the graph relationships?

Reply:
To test the graph relationships we need to use vector search to find the initial node and then traverse it.